tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1824,107 +1797,85 @@ def test_continuous_writes_during_pod_restart(self):
         assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
         assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
-        LOGGER.info("Step 8: Successfully completed Writes in background")
-
-        LOGGER.info("Step 9: Create multiple buckets and run IOs")
-        users = self.mgnt_ops.create_account_users(nusers=1)
-        self.s3_clean.update(users)
-        self.test_prefix = 'test-34075-1'
+        LOGGER.info("Step 5: Successfully completed Writes in background")
+        LOGGER.info("Step 6: Run READ/Verify on data written in healthy cluster")
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix,
-                                                    skipcleanup=True)
+                                                    log_prefix=self.test_prefix, skipwrite=True,
+                                                    skipcleanup=True, setup_s3bench=False)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 9: Successfully created multiple buckets and ran IOs")
-
+        LOGGER.info("Step 6: Read/Verify successful on data written in healthy cluster")
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Step 7: Create multiple buckets and run IOs")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.s3_clean.update(users)
+            self.test_prefix = 'test-34075-restart'
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
```

Comment:
This should be out of if so that IOs will be run on existing buckets in case of DTM enabled.

Reply:
done